History.md

```diff
@@ -0,0 +1,23 @@
+## 0.6.1 (November 29, 2018)
+
+* [#148] Deprecate the use of the hubspot rake tasks. Deprecating these tasks
+  includes deprecating the use of `Hubspot::Utils.dump_properties` and
+  `Hubspot::Utils.restore_properties`.
+
+[#148]: https://github.com/adimichele/hubspot-ruby/pull/148
+
+
+* [#148] Fix backwards compatibility to ensure users can access the hubspot rake
+  tasks
+
+[#148]: https://github.com/adimichele/hubspot-ruby/pull/148
```

Comment:
This updates the reference to point to PR #148 as discussed in this [comment](https://github.com/adimichele/hubspot-ruby/pull/160#discussion_r241513235)